enhancements/security/openshift-image-policy.md

```diff
@@ -0,0 +1,504 @@
+---
+title: openshift-image-policy
+authors:
+  - "@wking"
+reviewers:
+  - "@jupierce, updates staff engineer"
+  - "@mrunalp, node staff engineer"
+  - "@mtrmac, RHCOS, containers/image"
+  - "@petr-muller, updates"
+  - "@saschagrunert, node"
+  - "@zaneb, installer staff engineer"
+approvers:
+  - "@mrunalp, machine-config and node staff engineer"
+api-approvers:
+  - "@JoelSpeed"
+creation-date: 2024-05-29
+last-updated: 2024-05-31
+tracking-link:
+  - https://issues.redhat.com/browse/OCPSTRAT-1245
+see-also:
+  - "/enhancements/api-review/add-ClusterImagePolicy-and-ImagePolicy-for-Signature-Verification.md"
+---
+
+# OpenShift Image Policy
+
+## Summary
+
+[An existing enhancement](/enhancements/api-review/add-ClusterImagePolicy-and-ImagePolicy-for-Signature-Verification.md) is adding ClusterImagePolicy and ImagePolicy primitives to config image policy.
+This enhancement describes how those primatives will be used by OpenShift clusters to secure core workloads:
+
+* The `quay.io/openshift-release-dev/ocp-release` scope, used for OpenShift release images, will require [Sigstore][] signature signed by Red Hat, for any namespace in the cluster.
+* Users may add additional policies if they want, to add additional requirements as they see fit, including for the `quay.io/openshift-release-dev` scope.
+
+## Motivation
+
+As an OpenShift cluster administrator, I want to trust that the release image I'm running comes from Red Hat using the emerging industry standard for doing so: [Sigstore][].
+Sigstore is both cheaper to run (because it's less custom than the existing OpenPGP / GPG system) and more powerful (because it verifies signatures on every pod launch, not just during OpenShift updates), and both of those motivate our desire to transition.
```

Comment:
Makes sense.  I've pushed 49e25242 -> ecade8dc replacing this sentence with:

```Markdown
Sigstore is cheaper to run, because it's less custom as an image-signing mechanism than the existing OpenPGP / GPG system, and the ecosystem around generating, propagating, and verifying signatures is expected to be more robust.

CRI-O policies are more powerful than the cluster-version operator's OpenPGP verification, because they verifies signatures on every pod launch, not just during OpenShift updates.
[`containers-policy.json` supports OpenPGP requirements too][containers-policy-signedBy], but that property has not yet been exposed as an OpenShift configurable.
```

I'll mark this thread resolved,  but let me know if you think I still need text chances to capture this distinction :)